### ISSUE TITLE
HTTP 403 error when accessing Googe Drive data

### ISSUE THREAD

**Issue by ivmartel** [ivmartel/dwv#215]
Sometimes, the loading of Google Drive DICOM data fails with a [HTTP 403 error](https://en.wikipedia.org/wiki/HTTP_403). Refreshing fixes the problem but it can occur another time with different data.

**Comment by ivmartel** [comment]
Not able to reproduce with Google Chrome 51 or Firefox 47...

**Comment by ivmartel** [comment]
Postponing to next release.

**Comment by ivmartel** [comment]
Able to reproduce! Can it be related to new data (ie recently uploaded...)?

When clicking the failing link, I get: `The referrer null does not match the referrer restrictions configured on your API key. Please use the API Console to update your key restrictions.`

**Comment by ivmartel** [comment]
Seems to happen anytime... I just got it with previously uploaded data.

New, slightly different, message: `The request did not specify any referer. Please ensure that the client is sending referer or use the API Console to remove the referer restrictions.`

Switching to IP referer based restriction instead of website...

**Comment by ivmartel** [comment]
Ok, my mistake 😁 ... The error that I got when clicking the link is normal, there cannot be a `referer` when you click the request link that triggered the error...

The good way of doing it is to, in Google Chrome, switch the network tab on and refresh the page. That way I saw the error was related to quotas of queries per second. The code was sending one request per file. The solution is to use a [batch request](https://developers.google.com/api-client-library/javascript/reference/referencedocs#batch-api-requests) to group requests in one. See commit 0524cf5.

Referer is back to website with my github pages and the Google apis content one.
